lib/css/modules/_dashboard.scss

```diff
@@ -374,3 +374,21 @@ ul.widgetStateButtons {
 #userTaggerTable tr:hover .deleteIcon {
 	color: #369;
 }
+
+.res-r-dashboard {
```

Comment:
Actually, instead let's add `module.bodyClass = true;` in dashboard (if it's not already) and use `.res-dashboard`